app/src/main/contrib/actions/estimatesActionCreators.ts

```diff
@@ -64,10 +63,11 @@ export const estimatesActionCreators = {
                 data: true
             } as PopulatingEstimateSet);
 
-            const ids = mapStateToPropsHelper.getResponsibilityIds(getState());
+            const state = getState();
+            const ids = mapStateToPropsHelper.getResponsibilityIds(state);
 
             const result = await (new EstimatesService(dispatch, getState))
-                .populateEstimatesFromFile(ids.groupId, ids.touchstoneId, ids.scenarioId, ids.estimateSetId, uploadToken);
+                .populateEstimatesFromFile(ids.groupId, ids.touchstoneId, ids.scenarioId, state.estimates.populatingSetId, uploadToken);
```

Comment:
This wasn't actually affecting behaviour because this endpoint doesn't validate the set id by the looks of things, it just uses the upload token to identify the intended set. We should probably change this!